op-node/rollup/engine/engine_controller.go

```diff
@@ -397,6 +401,16 @@ func (e *EngineController) InsertUnsafePayload(ctx context.Context, envelope *et
 		})
 	}
 
+	totalTime := fcu2Finish.Sub(newPayloadStart)
+	e.log.Info("Inserted new L2 unsafe block (legacy)",
```

Comment:
Using same log message here (except for the `(legacy)` part) as in new codepath [here](https://github.com/ethereum-optimism/optimism/blob/020d4e31a9d7238aafed6c9c9b2b2702ba699146/op-node/rollup/engine/events.go#L361) so that we can easily filter and plot this log alongside the one in the new codepath

Reply:
Good call. Changed in https://github.com/ethereum-optimism/optimism/pull/12907/commits/026614d0f5b2c286a0ba552cbdfb16ae72dd6cca